### PR TITLE
Add google Pay value objects for generating XML

### PR DESCRIPTION
**Pull request by timcwebster**
Add google Pay value objects for generating XML for authorisation request

### CODE REVIEW

File: src/Inviqa/Worldpay/Api/Request/PaymentService/Submit/Authorisation/Order/ShopperBasic.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
 use Inviqa\Worldpay\Api\Request\PaymentService\Submit\Authorisation\Order\Shopper\ShopperEmailAddress;
 use Inviqa\Worldpay\Api\XmlNodeDefaults;
 
-class ShopperApplePay extends XmlNodeDefaults
+class ShopperBasic extends XmlNodeDefaults
```
Comment (ubick) [inline src/Inviqa/Worldpay/Api/Request/PaymentService/Submit/Authorisation/Order/ShopperBasic.php:8]: Why was ShopperApplePay renamed to ShopperBasic?
Why not use a separate ShopperGooglePay class?
Are they exactly the same?

**Review by ubick — CHANGES_REQUESTED** [review]
Great work. Just curious about the ShopperBasic class.

File: src/Inviqa/Worldpay/Api/Request/PaymentService/Submit/Authorisation/Order/ShopperBasic.php (hunk 1 above)
Comment (timcwebster) [reply src/Inviqa/Worldpay/Api/Request/PaymentService/Submit/Authorisation/Order/ShopperBasic.php:8]: yeah, Apple and Google use he exact same shopper class, so didn't see any point in creating 3 shopper classes

Shopper (for Worldpay)
ShopperApple
ShopperGoogle (same as Apple)

**Review by ubick — APPROVED** [review]
LGTM 👍

**Pull request merged**